docker/generated/bento-server/README.md

```diff
@@ -1,8 +1,10 @@
+<!-- P/S: even this file is autogenerated :happy: -->
+
 # Unified Model Serving Framework  [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=BentoML:%20The%20Unified%20Model%20Serving%20Framework%20&url=https://github.com/bentoml&via=bentomlai&hashtags=mlops,bentoml)
 
 BentoML is an open platform that simplifies ML model deployment and enables you to serve your models at production scale in minutes
 
-👉 [Pop into our Slack community!](https://join.slack.bentoml.org) We're happy to help with any issue you face or even just to meet you and hear what you're working on :)
+👉 [Pop into our Slack community!](https://l.linklyhq.com/l/ktOu) We're happy to help with any issue you face or even just to meet you and hear what you're working on :)
```

Comment:
can you confirm with Tim about the link to use here?

Reply:
@timliubentoml can you confirm that the link we use is correct?